index.html

```diff
@@ -12,7 +12,29 @@
 <body>
 
 
+	<table id="tab1" style="width:100%">
+	<tr>
+		<th>Name</th>
+		<th>Path</th>
+		<th>File Count</th>
+	</tr>
+	</table>
+
+	<form id=dataForm>
+		File name:<br>
```

Comment:
Używaj znaczników `<label>`
```suggestion
		<label for="id">File name:</label><br>
```